php_phongo.c

```diff
@@ -2016,6 +2016,18 @@ static void php_phongo_client_dtor(void *client)
 }
 #endif
 
+static void _phongo_check_json (void) __attribute__((constructor));
+static void _phongo_check_json (void)
```

Comment:
That's a problem then, as it would mean that PHP's normal loading reordering will no longer work. Basically, making the off case work (where distributions mangle PHP's original itents), but not the standard case where PHP itself manages the loading/symbol fetching order (the patch actively prevents this from working now).